src/data-pipeline/etl-common/src/main/resources/ts/traffic_source_channel_rule_v0.json

```diff
@@ -0,0 +1,389 @@
+[
+  {
+    "id": 1,
+    "channel": "Direct",
+    "condition": {
```

Comment:
Add display name and description: 
```
{
  displayName: {
    "zh-CN": string;
    "en-US": string;
  },
  description: {
    "zh-CN": string;
    "en-US": string;
  }
}
```

Reply:
added

Reply:
Use "zh-CN" & "en-US" as key, keep consistent with i18n.

Reply:
changed.